src/webview/helm-chart/app/cardItem.tsx

```diff
@@ -117,98 +117,166 @@ export class CardItem extends React.Component<DevFileProps, {
+                                                },
+                                                paddingLeft: '5px !important'
+                                            }
+                                        }}
+                                        style={{ marginLeft: '10px' }}
+                                        {...params}
+                                        inputProps={{
+                                            ...params.inputProps
+                                        }} />
+                                )}
+                                renderGroup={(params) => (
+                                    <li key={params.key}>
+                                        <this.GroupHeader>{params.group}</this.GroupHeader>
+                                        <this.GroupItems>{params.children}</this.GroupItems>
+                                    </li>
+                                )}
+                                onChange={(e, v) => this.setSelectedVersion(e, v)} />
                         </div>
                     </div>
-                    <div style={{ display: 'flex', flexDirection: 'row', gap: '0.5rem', width: '11rem', maxWidth: '15rem' }}>
-                        <InputLabel
-                            style={{
-                                color: '#EE0000',
-                                paddingTop: '0.5rem',
-                                marginLeft: '1rem'
-                            }}>
-                            Version:
-                        </InputLabel>
-                        <div
-                            data-testid='projects-selector'
-                            className={this.props.cardItemStyle.starterProjectSelect}
-                        >
-                            {this.state.versions.map((chart: Chart) => (
-                                <div
-                                    key={chart.version}
-                                    data-testid={`projects-selector-item-${chart.version}`}
-                                    onMouseDown={(): void => this.setSelectedVersion(chart)}
-                                    style={{width: '3rem', maxWidth: '6rem'}}
-                                    className={
-                                        selectedVersion.version === chart.version ? this.props.cardItemStyle.starterProjectSelected : this.props.cardItemStyle.project
-                                    }
-                                >
-                                    {chart.version}
-                                </div>
-                            ))}
+                    <div style={{ width: '50%', minHeight: '5rem', maxHeight: '10rem' }}>
+                        {selectedVersion.description &&
+                            <div className={this.props.cardItemStyle.detailedDescription}>
+                                <Typography variant='body1' className={this.props.cardItemStyle.helmCardDetailItem}>Description</Typography>
```

Comment:
```suggestion
                                <Typography variant='body1' style = {{ margin: '30px',  wordBreak: 'break-word'}}>Description</Typography>
```

Reply:
Also the css applied here has `margin: 0px`, we need to remove that, so that the description text below has some line difference. I see the css applied is coming from global reference.